GetStream/stream-chat-android
# [CAS-934] Create documentation for Client > Threads and Replies section

https://stream-io.atlassian.net/browse/CAS-934

### 🎯 Goal

Create Documentation for `Client > Threads and Replies` section

### 🎨 UI Changes

![screencapture-localhost-3000-docs-client-clientThreadAndReplies-2021-04-29-04_00_44](https://user-images.githubusercontent.com/9600921/116490236-8637c380-a89f-11eb-82ef-92f36ce83b17.png)

### 🧪 Testing

1. Go to documentation folder
2. Run `yarn run start`
3. Check out Client -> Threads and Replies section

### ☑️ Checklist


### 🎉 GIF

![giphy (1)](https://user-images.githubusercontent.com/9600921/116488534-3f47cf00-a89b-11eb-9757-34741fcfa425.gif)
